ubuntu_20.04-stable.dockerfile

```diff
@@ -69,6 +70,10 @@ ENV PYTHONPATH $HOME/opt/moab/lib/python3.8/site-packages/
 RUN mkdir dagmc \
     && cd dagmc \
     && git clone --branch develop --single-branch https://github.com/svalinn/DAGMC.git DAGMC \
+    && cd DAGMC \
+    && TAG=$(git describe --abbrev=0 --tags) \
+    && git checkout tags/`echo ${TAG}` -b `echo ${TAG}` \
```

Comment:
Since we clone only the `develop` branch, I'm not sure this has much effect.  Perhaps we should stick just naming the branch that we clone and make sure to get coordinated branches for DAGMC and PyNE